.circleci/config.yml

```diff
@@ -0,0 +1,31 @@
+version: 2
+jobs:
+  build:
+    machine:
+      image: circleci/classic:201710-02
+    working_directory: /home/circleci/nfm_tests
+    steps:
+      - checkout:
+          path: /home/circleci/nfm_tests
+      - run:
+          name: Change to Python 3.6 and install python dependencies
```

Comment:
says py 3.6

Reply:
i meant to say, testkraken depends on pydra and therefore py3.7+ 

perhaps there should be a difference between the environment in which the software is installed, and the environment where it's tested. should be ok to create a testkraken environment using conda for testing that is different from any runtime environment for the software.

Reply:
I will change to 3.7, have no idea why it's 3.6